custom/icds_reports/management/commands/run_custom_data_pull.py

```diff
@@ -0,0 +1,81 @@
+import os
+import zipfile
+
+from django.conf import settings
+from django.core.management.base import (
+    BaseCommand,
+    CommandError,
+)
+from django.db import connections
+
+from custom.icds_reports.const import CUSTOM_DATA_PULLS
+
+
+class Command(BaseCommand):
+    help = "Dump data from a pre-defined query for ICDS data pull requests"
+
+    def add_arguments(self, parser):
+        parser.add_argument('name')
```

Comment:
Would be good to document this param as it's probably the most interesting and non-obvious one

Reply:
https://github.com/dimagi/commcare-hq/pull/26304/commits/7462b24f046acaed74e90a3f9a650df0b5781d44